### PR TITLE
removal issue: code ref, test, marking animations updated

### PR DESCRIPTION
**Pull request by vaqueraoscar0**
## Code Ref details
- Cypress test (expand-arrow) fixed.
- Some code was re-structured and cleaned up
- marking removal issue resolved
- some css animation were adjusted
- Special case for single suggestion request added

### CODE REVIEW

File: src/app/home/home.component.css [hunk 1]
```diff
@@ -51,10 +51,10 @@
 }
 
 .fade-out {
-    animation: fadeOutAndSlideRight 1s cubic-bezier(0.42, 0, 0.58, 1);
+    animation: fadeout-And-Slide-Right 1s cubic-bezier(0.42, 0, 0.58, 1);
```
Comment (AndiBraimllari) [inline src/app/home/home.component.css:54]: or perhaps even `fade-out-and-slide-right`?

File: src/app/home/home.component.css [hunk 2]
```diff
@@ -81,12 +81,12 @@
 }
 
 .move-up-multiple-animation {
-    animation: moveUpMultiple 0.8s cubic-bezier(0.42, 0, 0.58, 1);
+    animation: move-Up-Multiple 0.8s cubic-bezier(0.42, 0, 0.58, 1);
 }
 
-@keyframes moveUpMultiple {
+@keyframes move-Up-Multiple {
```
Comment (AndiBraimllari) [inline src/app/home/home.component.css:87]: perhaps `move-up-multiple`?

File: src/app/home/home.component.ts [hunk 3]
```diff
@@ -1209,16 +1217,11 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
         }, 2 * this.SECONDS);
     }
 
-    private screenHeightAnimation(action: string) {
+    private removeScreenHeightDelay() {
```
Comment (AndiBraimllari) [inline src/app/home/home.component.ts:1220]: Don't forget to add the return type for this method.

**Comment by AndiBraimllari** [comment]
Overall changes look pretty neat I'd say. Could you have a look at this issue, https://github.com/OpenCovenant/quill/issues/368

I feel like it might be something small, if so perhaps you can do it in this PR, if not then feel free to work on it in another PR and we can merge this one.

**Comment by vaqueraoscar0** [comment]
Can you clarify that for me. I read it i just didn't quite know what you were referring to.

File: src/app/home/home.component.html [hunk 4]
```diff
@@ -495,16 +495,22 @@
...
-            <ul class="list-group list-group-flush while-reading-text-ul"
+            <ul
+                class="list-group list-group-flush while-reading-text-ul"
                 [ngClass]="{
-                    'while-reading-text-ul-dark-mode': darkModeService.isDarkMode
-                }">
+                    'while-reading-text-ul-dark-mode':
+                        darkModeService.isDarkMode
+                }"
+            >
```
Comment (AndiBraimllari) [inline src/app/home/home.component.html:513]: Do these changes serve any purpose? If it is just for indentation, I'd say undo these changes and only push full-file formatting changes e.g. with prettier in a separate PR if you'd like.

Comment (vaqueraoscar0) [reply src/app/home/home.component.html:513]: it's a formatting issue, that was resolved by itself after running the prettier:fix method.

Comment (vaqueraoscar0) [reply src/app/home/home.component.html:513]: ok i will
